www/kanban/inner.js

```diff
@@ -534,6 +536,137 @@ define([
         UI.openCustomModal(editModal.modal);
     };
 
+    addMoveElementButton = function (framework, kanban) {
+        var $container = $(kanban.element);
+        var drag = $('#toggle-drag-off').attr('class').indexOf('toggle-active') !== -1 ? false : true;
+        kanban.options.dragBoards = drag;
+        kanban.options.dragItems = drag;
+        if (drag === false) {
+            if (!kanban) { return; }
```

Comment:
If kanban is falsy, this code will throw an error 5 lines above with `var $container = $(kanban.element);`